app/assets/stylesheets/pages.css.scss

```diff
@@ -129,12 +129,21 @@ $include-html-global-classes: false;
 .publications {
   background: $hackedu-black;
   color: $text-on-black;
+  padding: 5px 0 5px 0;
+
+  @media #{$medium-up} {
+    padding: 25px 0 25px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `25px 0` instead of `25px 0 25px 0`